library/src/components/Schema.tsx

```diff
@@ -300,7 +310,12 @@ export const Schema: React.FunctionComponent<Props> = ({
                   <Schema
                     key={idx}
                     schema={s}
-                    schemaName={idx === 0 ? 'Consists of:' : 'And with:'}
+                    schemaName={SchemaHelpers.applicatorSchemaName(
```

Comment:
wouldn't it be better if `applicatorSchemaName` just accepts 2 arguments?

`SchemaHelpers.applicatorSchemaName(idx, s)` or `SchemaHelpers.applicatorSchemaName(idx, s.title())` as testing will be easier

Reply:
What would be better/easier exactly 🤔? I don't see how we make this work without either: 

- duplicating the pattern-match logic on the schema applicator kind inside the `applicatorSchemaName` function.

- or, duplicating the title creation logic on the index number at every call site. 

Do you have a more concrete example of what you have in mind?

Reply:
oh crap, forget it, sorry for monday blindness... I seriously did not notice the other usage of `applicatorSchemaName` usage 🤦🏼